### PR TITLE
Format the README style

### PR DESCRIPTION
**Pull request by yungyuc**
This PR only fixes the typesetting of README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +57,8 @@ more flexibility and saves significant memory.
 
 ### Running Unitest
 
-We use `shunit2` as our unittest framework. If you want to enable unittest, please fetch your source along with `shunit2` by placing:
+We use `shunit2` as our unittest framework. If you want to enable unittest,
```
Comment (yungyuc) [inline README.md:60]: Break the long line into 2.  Note that at the end of the file (line 74) the textwidth has been indicated to be 79.

File: README.md [hunk 2]
```diff
@@ -17,7 +17,6 @@ following features:
 The environment consists of scripts for building dependencies and managing
 them.  More information will be added as the project becomes more complete.
 
-
```
[the remark this answers is not in the document]
Comment (tai271828) [reply README.md:20]: It seems my own convention. :)

[the remark this answers is not in the document]
Comment (tai271828) [reply README.md:20]: I should communicate with the team first.

**Review by tai271828 — APPROVED** [review]
Looks good me. We could talk about the markdown convention style later in the sprint.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (yungyuc) [reply README.md:20]: Could you elaborate more about the convention you use?  I did not see a pattern but I can be wrong.

[the remark this answers is not in the document]
Comment (tai271828) [reply README.md:20]: Let me initiate a discussion for the elaboration https://github.com/solvcon/devenv/discussions/60

**Pull request merged**